Subject: [PATCH] Fix the momery leak in DrawOverlayDetails

https://code.google.com/p/android/issues/detail?id=210852

we need to release the mOverlayBridge in onDestroy, or it will cause Memory Leak issue

diff --git a/src/com/android/settings/applications/DrawOverlayDetails.java b/src/com/android/settings/applications/DrawOverlayDetails.java
index ef92c21..de27cde 100644
--- a/src/com/android/settings/applications/DrawOverlayDetails.java
+++ b/src/com/android/settings/applications/DrawOverlayDetails.java
@@ -88,6 +88,12 @@
     }
 
     @Override
+    public void onDestroy() {
+        super.onDestroy();
+        mOverlayBridge.release();
+    }
+
+    @Override
     public boolean onPreferenceClick(Preference preference) {
         if (preference == mOverlayPrefs) {
             if (mSettingsIntent != null) {
